include/rocksdb/c.h

```diff
@@ -1119,7 +1119,11 @@ enum {
   rocksdb_total_metric_count = 68
 };
 
+#include "perf_flag_defs.h"
```

Comment:
is it necessary?

Reply:
yes, but not that much, It can be removed but in perf level is also included, thats whay I included.